src/masonite/validation/Validator.py

```diff
@@ -1029,6 +1029,128 @@ def negated_message(self, attribute):
         return "The {} is a valid {} postal code.".format(attribute, self.locale)
 
 
+class different(BaseValidation):
+    """The field under validation must be different than an other given field."""
+    def __init__(self, validations, other_field, messages={}, raises={}):
+        super().__init__(validations, messages=messages, raises=raises)
+        self.other_field = other_field
+
+    def passes(self, attribute, key, dictionary):
+        other_value = dictionary.get(self.other_field, None)
+        return attribute != other_value
+
+    def message(self, attribute):
+        return "The {} value must be different than {} value.".format(attribute, self.other_field)
+
+    def negated_message(self, attribute):
+        return "The {} value be the same as {} value.".format(attribute, self.other_field)
+
+
+class uuid(BaseValidation):
+    """The field under validation must be a valid UUID. The UUID version standard
+    can be precised (1,3,4,5)."""
+    def __init__(self, validations, version=None, messages={}, raises={}):
+        super().__init__(validations, messages=messages, raises=raises)
+        self.version = version
+        self.uuid_type = "UUID"
+        if version:
+            self.uuid_type = "UUID {0}".format(self.version)
+
+    def passes(self, attribute, key, dictionary):
+        from uuid import UUID
+        try:
+            uuid_value = UUID(str(attribute))
+            if self.version:
+                return uuid_value.version == int(self.version)
+            else:
+                return True
+        except ValueError:
+            return False
+
+    def message(self, attribute):
+        return "The {} value must be a valid {}.".format(attribute, self.uuid_type)
+
+    def negated_message(self, attribute):
+        return "The {} value must not be a valid {}.".format(attribute, self.uuid_type)
+
+
+class required_if(BaseValidation):
+    """The field under validation must be present and not empty only
+    if an other field has a given value."""
+    def __init__(self, validations, other_field, value, messages={}, raises={}):
+        super().__init__(validations, messages=messages, raises=raises)
+        self.other_field = other_field
+        self.value = value
+
+    def passes(self, attribute, key, dictionary):
+        if dictionary.get(self.other_field, None) == self.value:
+            return required.passes(self, attribute, key, dictionary)
+        else:
```

Comment:
get rid of the else and just return

Reply:
What if `dictionary.get(self.other_field, None) == self.value` is False ? We won't return anything from passes() method ?

Reply:
I'm just saying do this:

```python
if dictionary.get(self.other_field, None) == self.value:
    return required.passes(self, attribute, key, dictionary)

return True
```

I hate those little dangling elses. Your code and the code snippet above are the same but it gets rid of the else keyword

Reply:
Oh yes sorry 😐

Reply:
It's corrected.